### PR TITLE
coq-ext-lib: add opam package for version 0.11.3

### PR DESCRIPTION
**Pull request by MSoegtropIMC**
From Coq Platform v8.13

FYI: @gmalecha

### CODE REVIEW

**Comment by palmskog** [comment]
@MSoegtropIMC can you explain why you're using a Git URL here? There is a perfectly functional tarball for this project: https://github.com/coq-community/coq-ext-lib/archive/v0.11.3.tar.gz

**Comment by MSoegtropIMC** [comment]
@palmskog : sorry I didn't create this package originally - I was merging stuff from Coq Platform. I will review them better next time (just checked that they lint and work).

File: released/packages/coq-ext-lib/coq-ext-lib.0.11.3/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "gmalecha@gmail.com"
+homepage: "https://github.com/coq-community/coq-ext-lib"
+dev-repo: "git+https://github.com/coq-community/coq-ext-lib.git"
+bug-reports: "https://github.com/coq-community/coq-ext-lib/issues"
+authors: ["Gregory Malecha"]
+license: "BSD"
```
Comment (palmskog) [inline released/packages/coq-ext-lib/coq-ext-lib.0.11.3/opam:7]:
```suggestion
license: "BSD-2-Clause-FreeBSD"
```
@liyishuai can you confirm that this is the intended license? https://spdx.org/licenses/BSD-2-Clause-FreeBSD.html

Comment (liyishuai) [reply released/packages/coq-ext-lib/coq-ext-lib.0.11.3/opam:7]: Yes, it is. I'll update the coq-ext-lib repo to use this standardized identifier accordingly.

**Comment by MSoegtropIMC** [comment]
@palmskog : this should be good to go now.

**Pull request merged**